<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransShiftClass</class>
 <widget class="QMainWindow" name="TransShiftClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TransShift</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="BTN_LOAD_IMG">
          <property name="text">
           <string>Load Damaged Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EDIT_PATH">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="BTN_LOAD_PATCH">
          <property name="text">
           <string>Load Patch Images</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="TBL_LIST">
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Path</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_START">
          <property name="text">
           <string>Run Restore Image</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>ratio_threshold</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="SPIN_RATIO">
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.700000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>ransac_iter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="SPIN_ITER">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>ransac_thresh</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="SPIN_THRESH">
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="EDIT_LOG">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_SAVE">
          <property name="text">
           <string>Save Result</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TransShift.qrc"/>
 </resources>
 <connections/>
</ui>
